test/unit/lib/censorBirthV1.js

```diff
@@ -0,0 +1,284 @@
+    sex: "Male",
+    birthplace: "Bath"
+  },
+  mother: {
+    prefix: "Ms",
+    forenames: "MOTHER FORENAME",
+    surname: "MOTHER SURNAME",
+    suffix: "N/A",
+    birthplace: "Exeter",
+    occupation: "Teacher",
+    aliases: [],
+    address: "ADDRESS 1",
+    maidenSurname: "MAIDEN SURNAME",
+    marriageSurname: "MARRIAGE SURNAME"
+  },
+  father: {
+    prefix: "MR",
+    forenames: "FATHER FORENAME",
+    surname: "FATHER SURNAME",
+    suffix: "ESQ",
+    birthplace: "BRISTOL",
+    occupation: "Cleaner",
+    aliases: [],
+    deceased: false
+  },
+  dateOfDeclaration: "2020-10-23",
+  dateOfStatutoryDeclarationOfParentage: "2020-10-23",
+  statutoryDeclarationOfParentage: "",
+  dateOfNameUpdate: "2020-10-24",
+  status: {
+    blocked: false,
+    marginalNote: "MARGINAL NOTE",
+    cancelled: false,
+    correction: "None",
+    nameUpdate: "None",
+    onAuthorityOfRegistrarGeneral: false,
+    potentiallyFictitious: false,
+    praOrCourtOrder: "None",
+    reregistration: "None"
+  },
+  previousRegistration: undefined,
+  nextRegistration: undefined
+}
+
+const redactedBirthDataNotBlocked = {
+  id: 12345,
+  date: "2020-09-23",
+  entryNumber: null,
+  registrar: {
+    signature: null,
+    designation: null,
+    superintendentSignature: null,
+    superintendentDesignation: null,
+    subdistrict: "Subdistrict town",
+    district: "District city",
+    administrativeArea: "Adminshire"
+  },
+  informant1: {
+    forenames: null,
+    surname: null,
+    address: null,
+    qualification: "Confidential Informant",
+    signature: null,
+    signatureIsMark: null
+  },
+  informant2: {
+    forenames: null,
+    surname: null,
+    address: null,
+    qualification: "Not a Confidential Informant",
+    signature: null,
+    signatureIsMark: null
+  },
+  child: {
+    originalPrefix: null,
+    prefix: null,
+    forenames: "Expensive",
+    originalForenames: null,
+    surname: "Mistake",
+    originalSuffix: null,
+    suffix: null,
+    dateOfBirth: "2020-09-23",
+    sex: "Male",
+    birthplace: "Bath"
+  },
+  mother: {
+    prefix: null,
+    forenames: "MOTHER FORENAME",
+    surname: "MOTHER SURNAME",
+    suffix: null,
+    birthplace: "Exeter",
+    occupation: null,
+    aliases: [],
+    address: null,
+    maidenSurname: "MAIDEN SURNAME",
+    marriageSurname: "MARRIAGE SURNAME"
+  },
+  father: {
+    prefix: null,
+    forenames: "FATHER FORENAME",
+    surname: "FATHER SURNAME",
+    suffix: null,
+    birthplace: "BRISTOL",
+    occupation: null,
+    aliases: [],
+    deceased: null
+  },
+  dateOfDeclaration: null,
+  dateOfStatutoryDeclarationOfParentage: null,
+  statutoryDeclarationOfParentage: null,
+  dateOfNameUpdate: null,
+  status: {
+    blocked: false,
+    marginalNote: "MARGINAL NOTE",
+    cancelled: false,
+    correction: "None",
+    nameUpdate: "None",
+    onAuthorityOfRegistrarGeneral: false,
+    potentiallyFictitious: false,
+    praOrCourtOrder: "None",
```

Comment:
Some of these can probably be safely redacted:
- nameUpdate
- onAuthorityOfRegistrarGeneral
- praOrCourtOrder
But please check which ones we are currently building flags from.

Reply:
OK - seems you're right, these 3 fields aren't required.

Reply:
Worth checking the others but I think we need them.

Reply:
Yeah I think you are right - removed the 3 you have suggested